fedora-toolbox

```diff
@@ -246,6 +250,66 @@ enter()
 )
 
 
+list_images()
+(
+    output=$($prefix_sudo podman images \
+            --filter "label=com.redhat.component=fedora-toolbox" \
+            --format "{{.ID}} {{.Repository}}:{{.Tag}} {{.Created}}" \
+            | sed 's/ \{2,\}/\t/g' | column --separator $'\t' --table \
+            --table-columns "IMAGE ID,IMAGE NAME,CREATED")
+    if ! [[ $output = "" ]]; then
+        echo -e "${LBC}Images created by fedora-toolbox${NC}\n$output"
+    fi
+)
+
+
+list_containers()
+(
+    output=$($prefix_sudo podman ps \
+            --all \
+            --filter "label=com.redhat.component=fedora-toolbox")
+            #--format "table {{.ID}} {{.Names}} {{.Created}} {{.Status}} {{.Image}}")
+    if ! [[ $(echo "$output" | wc -l) -lt 2 ]]; then
+        to_print="${LBC}Containers created by fedora-toolbox${NC}\n"
+        while read -r line; do
+            if [[ $line = "" ]]; then
+                break
+            fi
+            line_id="$(cut -d' ' -f1 <<< $line)"
+            if ! [[ $line_id = "CONTAINER" ]]; then
+                inspect_out=$($prefix_sudo podman inspect $line_id --format "{{.State.Running}}")
+            fi
+            if [[ $inspect_out = "true" ]]; then
+                to_print="$to_print${LGC}$line${NC}\n"
+            else
+                to_print="$to_print$line\n"
+            fi
+        done <<< "$output"
```

Comment:
Sadly, the `<<<` construct isn't in POSIX, so we need to pipe the string into the `while` loop.

Reply:
Done.